dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM ubuntu:20.04
+
+RUN useradd -M pgbouncer && \
+    apt-get -y update && \
+    apt-get -y install pgbouncer && \
+    rm -rf /var/lib/apt/lists/*
+
+WORKDIR /etc/pgbouncer
+COPY ./pgbouncer.ini pgbouncer.ini
+COPY ./userlist.txt userlist.txt
```

Comment:
I think we should drop this and generate on first run based on environment variables. Will follow up with you! :)

Reply:
I've added an entrypoint script that should create acceptable basic config files, but I'd imagine pgbouncer.ini can get a lot more complicated than what I've implemented. I'm looking to find some example pgbouncer.ini files so I can better understand how to implement this.